implementations/rust/ockam/ockam_api/src/control_api/openapi.rs

```diff
@@ -24,12 +26,93 @@ impl Modify for Authentications {
     }
 }
 
+const MAIN_DESCRIPTION: &str = r#"
+## Overview
+This API is designed to control Ockam nodes via HTTP requests without having to use Ockam Command or Ockam library.
+You can expect a similar abstraction level as Ockam Command, but with a more programmatic approach.
+
+The endpoint will receive the HTTP request and forward it to the selected node using the `node`
+parameter. How the node will be selected starting from the name is configuration dependent.
+
+## Versioning
+Current major `0.Y.Z` is considered unstable and may have breaking changes.
+Future versions will follow semantic versioning, breaking changes will be reflected in the major version number.
+
+## Authentication
+Only a simple bearer token is required to authenticate. The token is passed in the `Authorization`
+header: `Authorization: Bearer my-secret-token`.
+
+## Development
```

Comment:
I think that the development section should be left aside in a Markdown doc for developers, since this is the API description for its users.

Reply:
Mhmm In my mind whoever is reading the API documentation is a developer.
I'll rename the section from "Development" into "Getting started" to avoid confusion